ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -1410,7 +1410,16 @@ namespace OrdinalAxis {
 
             // Check whether the series has at least one point inside the chart
             const hasPointsInside = function (series: Series): boolean {
-                return series.points.some((point): boolean => !!point.isInside);
+                return series.points.some(function (point): boolean {
+                    let hasPointsInside = false;
+
+                    if (axis.min && axis.max) {
```

Comment:
Nitpicking: move this condition outside of the `some()` method. There's no need to check this condition for every point, right?

Reply:
That's a good point, thank you!

I have also simplified the code to improve the readability.

Reply:
That's a good point, thank you

I have also simplified the code to improve readability.

Reply:
Also, I have added the `isNumber` check, since if `axis.min` would be 0, previously it would fail.